src/Compilers/Core/Portable/Operations/Queries/IQueryContinuation.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents a query continuation in C#.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface IQueryContinuation : IOperation
+    {
+        /// <summary>
+        /// Declared symbol.
+        /// </summary>
+        IRangeVariableSymbol DeclaredSymbol { get; }
```

Comment:
>IRangeVariableSymbol DeclaredSymbol { get; } [](start = 8, length = 44)

Many query clauses declare range variables, but it looks like only IQueryContinuation exposes this information. I would expect, that either all relevant nodes would expose this information, or none.